config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false #KIRSTIN for Docker
```

Comment:
What is the advantage of changing that parameter to false for development? It's new to me and I didn't know what it changes without doing a quick web search. That told me by changing this value to false, we might not get the useful error report whenever an exception occurred. Instead, and I proved that on my machine, it will display a 'user-friendly' error message and hide more details (e.g. page does not exist or internal application error).